Content.Shared/Body/Systems/SharedBodySystem.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Content.Shared.Damage;
+using Content.Shared.Standing;
+using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Body.Systems;
+
+[Virtual]
```

Comment:
no need attribute if its abstact

Reply:
Fixed in [`f686967` (#11991)](https://github.com/space-wizards/space-station-14/pull/11991/commits/f686967a43b0fecad760fa5af9b9581483f82690)